client/state/form/packages/reducer.js

```diff
@@ -8,7 +8,8 @@ import {
 	TOGGLE_OUTER_DIMENSIONS,
 } from './actions';
 import omitBy from 'lodash/omitBy';
-import isNull from 'lodash/isNull';
+
+const isNullOrEmpty = ( value ) => null === value || '' === value;
```

Comment:
I think we should trim whitespace before the empty part of this check.

Putting a single space character in the outer dimensions field still results in a value being set.